Gemfile

```diff
@@ -3,13 +3,20 @@ source 'https://rubygems.org'
 gemspec
 
 group :development do
-  gem 'vagrant', git: 'https://github.com/mitchellh/vagrant.git'
+  gem 'vagrant',      git: 'https://github.com/mitchellh/vagrant.git'
+  gem 'vagrant-spec', git: 'https://github.com/mitchellh/vagrant-spec.git'
+  gem 'gem-compare'
+  gem 'rake'
+  gem 'vagrant-vbguest'
+  gem 'vagrant-libvirt'
```

Comment:
> This is also not compulsory for Virtualbox provider

what is not? Note, these gems are in the development group, so there is no harm for the actual runtime dependencies.